luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/LuthierIndustrialPark.java

```diff
@@ -81,9 +92,10 @@ public Dimension getDimension(String dimensionName) {
     }
 
     /**
-     * Builds a SearchProvider for a specific domain.
+     * Retrieve or build a SearchProvider for a specific domain.
      *
      * @param domain  a string that is associated with the type o
```

Comment:
Not strictly related to this PR, but it looks like the documentation here was never finished.

Reply:
Sometimes I wish the styler linter can semantically check whether my comments are finished...

Good catch! Fixed in this PR.